### ISSUE TITLE
All Pages - Missing ALT Text/Text for "Worker.gov" Logo

### ISSUE THREAD

**Issue by jfc3-dol** [presidential-innovation-fellows/worker-dol-gov#6]
While testing using the DHS Trusted Tester process, I found there is no text/ALT text for the "Worker.gov" logo.

Section 508 - 22(a)
WCAG 2.0 - 2.2 Web: Images

**Comment by adamsfallen** [comment]
Fixed - 4457011dc304bdb7f0aa46739d2195c1a3e8df31

**Comment by jfc3-dol** [comment]
Thanks, for making the quick update for this issue and the other couple.

There is no need for the "Site logo" part of the ALT text because the image is part a link to the home page of the website. Now if the worker.gov logo was on a page about the logo and wasn't a link adding the word "logo" to the ALT text, makes sense.

I think we need to open this issue back up to correct the new issue.

In the future can I please look at the code updates before closing the issue?

**Comment by adamsfallen** [comment]
What would you prefer the alt text to read?

**Comment by jfc3-dol** [comment]
"Worker.gov" works for me if that is the only name we have for the website.

**Comment by adamsfallen** [comment]
Updated - cb33fb46355a3004e28975c5f599cf1bb7ad352e - FYI I'll leave it to you to accept / close issues for all of these (including this one).

**Comment by adamsfallen** [comment]
@jfc3-dol does this work for you?

**Comment by jfc3-dol** [comment]
Yes, it does work for me. Closing issue.
